cmd/gapic-showcase/compliance_suite_test.go

```diff
@@ -217,6 +217,16 @@ func prepRepeatDataPathResourceTest(request *genproto.RepeatRequest) (verb strin
 	name = "Compliance.RepeatDataPathResource"
 	info := request.GetInfo()
 
+	if strings.HasPrefix(info.GetFString(), "first/") {
+		return prepRepeatDataPathResourceTestFirstBinding(request)
```

Comment:
(no action needed) I think this is fine, but it makes me think in the future we should change the `first/` and `second/` in the URL templates to other names for clarity, like `blue` and `cyan` or something.....